<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectMeters4pollTopWdgt</class>
 <widget class="QWidget" name="SelectMeters4pollTopWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblDatabase">
     <property name="text">
      <string>Database: processing...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbPollAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>All visible</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
         <normaloff>:/katynko/svg/table.svg</normaloff>:/katynko/svg/table.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSelected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Selected</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
         <normaloff>:/katynko/svg2/sx12452-all.svg</normaloff>:/katynko/svg2/sx12452-all.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCheckedOn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Checked &quot;On&quot;</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
         <normaloff>:/katynko/svg/flag-red.svg</normaloff>:/katynko/svg/flag-red.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
         <normaloff>:/katynko/svg/edit-delete.svg</normaloff>:/katynko/svg/edit-delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Ignored</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbxGroups">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxIgnoreExistingData">
     <property name="text">
      <string>Ignore existing data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Matilda-units/gui/image-and-font/res.qrc"/>
 </resources>
 <connections/>
</ui>
